oak_abi/src/lib.rs

```diff
@@ -44,6 +44,16 @@ pub const OAK_LABEL_HTTP_JSON_KEY: &str = "oak-label";
 /// The header key used for protobuf encoded Labels in HTTP requests.
 pub const OAK_LABEL_HTTP_PROTOBUF_KEY: &str = "oak-label-bin";
 
+/// The header key used for JSON formatted signed authentication challenge.
+pub const OAK_SIGNED_CHALLENGE_JSON_KEY: &str = "oak-signed-auth-challenge";
+
+/// The header key used for protobuf formatted signed authentication challenge.
+pub const OAK_SIGNED_CHALLENGE_PROTOBUF_KEY: &str = "oak-signed-auth-challenge-bin";
```

Comment:
Are we going to keep maintaining both JSON and protobuf versions for all headers going forward, or do you think we should coalesce them at some point? @rbehjati @juliettepretot